chore: update context/options (#1186)
keymanapp/help.keyman.com: developer/17.0/context/options.md

![Options dialog - Server tab](/cdn/dev/img/developer/170/ui/frmOptions_Server.png)

Keyman Developer Server
:   ![Keyman Developer Server Options dialog](/cdn/dev/img/developer/170/ui/frmKeyman_Developer_Server_Options.png)   
    

>     Testing out custom keyboards on a server with the new configuration of Keyman Develper Server will only enhanced the usability of the keyboard, with many features including:

Reviewer: Good update. I suggest rephrasing like:

```suggestion
    You can customize the Keyman Developer server for testing the usability of your keyboard. Features of the Keyman Developer server include:
```

    * Seamless integration with the IDE (Start, Stop, Live reload, Recompile...)
    * Port configuration
    * Allow testing cross devices
    * Wider range support for NGROK
    
    [Find out more](https://github.com/keymanapp/keyman/pull/6073)

List local URLs for Server
:   Untick the box to close showing any local URLs for keyboard testing

>     ![Keyman Developer Server Options dialog](/cdn/dev/img/developer/170/ui/frmList_Local_URLs.png) 

Reviewer: I'm unclear where this section is from the `frmKeyman_Developer_Server_Options.png` screenshot.

Maybe you can add more content about the tickboxes for:

Leave server running after closing IDE
Use ngrok to provide public url for web debugger
Show server console window on start

Author: I added the full length of information for the instructions above, Darcy. This section is related to another configuration of Keyman Developer Server here:
![image](https://github.com/keymanapp/help.keyman.com/assets/90595388/ee23615f-fded-4287-8849-c1093c86468a)

List local URLs for Server
:   Below Configure Server, untick the box, and it will quit displaying any local URLs for keyboard testing.
    ![Keyman Developer Server Options dialog](/cdn/dev/img/developer/170/ui/frmList_Local_URLs.png) 

> Explore more: [Keyman Developer Server](http://localhost:8055/developer/17.0/context/server#toc-configuring-keyman-developer-server)

Reviewer: Can we make this a relative path? See if this works
```suggestion
Explore more: [Keyman Developer Server](server#toc-configuring-keyman-developer-server)
```
